job-simulation.rb

```diff
@@ -1,4 +1,67 @@
 require './Stack.rb'
 require './Queue.rb'
 
+waiting = Queue.new
+workers = Stack.new
 
+puts "How many people do you have? (Must be greater than 6)"
+initial_num = gets.chomp.to_i
+
+1.upto(initial_num) do |n|
+  waiting.enqueue(rand(100..999))
+end
+
+puts "\nInitial Waiting List #{waiting.store}"
+
+new_hires = waiting.dequeue(6)
+
+new_hires.each do |new_hire|
+  workers.push(new_hire)
+end
+
+puts "\nWorkers List\n" +
+"1st hired: #{workers.store[0]}\n" +
+"2nd hired: #{workers.store[1]}\n" +
+"3rd hired: #{workers.store[2]}\n" +
+"4th hired: #{workers.store[3]}\n" +
+"5th hired: #{workers.store[4]}\n" +
+"6th hired: #{workers.store[5]}\n"
```

Comment:
This is good, but could you have planned for the potential to have a different number of workers by using a loop instead of hard coding these?